Makefile

```diff
@@ -50,11 +51,11 @@ test: generate fmt vet manifests
 
 # Build manager binary
 manager: generate fmt vet
```

Comment:
Should this be `operator` instead? It's a bit weird to build `./cmd/operator` into a binary with another name.

Reply:
agree, will need to shift a few other things around for consistency